device.mk

```diff
@@ -67,6 +67,15 @@ PRODUCT_PROPERTY_OVERRIDES += \
     ro.carrier=unknown \
     persist.vehicle.use-vis-hal=false \
 
+# Define audio tracks that we want to use as default.
+# Pay attention that this override has to be used after
+# include of car.mk in order to take precedence.
+PRODUCT_PROPERTY_OVERRIDES := \
```

Comment:
That was very first option that I did, but it didn't work.
So, I found discussion on google group with clarification from googler, that in case of PRODUCT_PROPERTY_OVERRIDES only very first declaration is used, and all others are ignored. That's why I need to put my strings before already existing.
Also I took into account that car.mk defines it's own ro.config.ringtone that is not available in our audio package. And car.mk use exactly same method - prepend but not append.
That's why I put in comment that these lines must be used after car.mk.

Reply:
Source https://groups.google.com/d/msg/android-building/Y8Z1hWqL0lY/1NytPiDGg9oJ

Reply:
Yes, I get it. Maybe it is a good idea to add a note like " The assignment should remain to guarantee this change will..".

Reply:
Agree. I will update commit with explanation in comment to avoid mistakes during changes in future.

Reply:
Comment reworked.